java/client/src/main/java/glide/api/commands/HashCommands.java

```diff
@@ -45,4 +45,20 @@ public interface HashCommands {
      *     If <code>key</code> does not exist, it is treated as an empty hash and it returns 0.<br>
      */
     CompletableFuture<Long> hdel(String key, String[] fields);
+
+    /**
+     * Returns all fields and values of the hash stored at <code>key</code>.
+     *
+     * @see <a href="https://redis.io/commands/hgetall/">redis.io</a> for details.
+     * @param key The key of the hash.
+     * @return A <code>Map</code> of fields and their values stored in the hash. Every field name in
+     *     the list is followed by its value. <br>
```

Comment:
`* @return A <code>Map</code> of fields and their values stored in the hash. Every field name in the map is associated with its corresponding value. <br>`